frontend/dockerfile/instructions/commands_runmount.go

```diff
@@ -176,6 +176,14 @@ func parseMount(value string) (*Mount, error) {
 			}
 			m.ReadOnly = !rw
 			roAuto = false
+		case "required":
+			if m.Type == "secret" || m.Type == "ssh" {
+				v, err := strconv.ParseBool(value)
+				if err != nil {
+					return nil, errors.Errorf("invalid value for %s: %s", key, value)
+				}
+				m.Required = v
+			}
```

Comment:
should there be an error if `required` is used for the mount type which doesn't support it?

Reply:
It could. It didn't before for the single value one so I didn't want to change the behavior.

Reply:
added second commit for this